python/svix/api.py

```diff
@@ -1074,7 +1074,7 @@ async def create(
             with_content=False,
             **options.to_dict(),
         )
-        ret.payload = MessageOutPayload.from_dict(message_in.payload.to_dict())
+        ret.payload = message_in.payload
```

Comment:
```suggestion
        ret.payload = t.Cast(message_in.payload, t.Any)
```
Something like this?